index.js

```diff
@@ -0,0 +1,133 @@
+const express = require('express')
+const teams = require('./teams.json')
+
+const app = express()
+
+
+
+app.get('/teams', (request, response) => {
+
+    response.send(teams)
+
+})
+
+app.get('/teams/', (request, response) => {
+
+    const allTeams = teams.map((team) => {
+
+        return team
+
+    })
+})
```

Comment:
Not sure what this route is for. It loops over all the teams making a new array that is exactly the same as the first array so effectively this route is the same as the one above it.